lisa/features/disks.py

```diff
@@ -62,7 +62,60 @@ class DiskType:
     DISK_EPHEMERAL: str = DiskEphemeral.name()
     DISK_PREMIUM: str = DiskPremiumLRS.name()
     DISK_STANDARD: str = DiskStandardLRS.name()
+    # refer https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types
+    DISK_SIZE_PREMIUM: Dict[str, int] = {
```

Comment:
`DISK_SIZE` and `DiskType` represent two different thing. Maybe we can move `DISK_SIZE` to their specific class

Reply:
fix